Updated Map section (#25726)
DataDog/documentation: content/en/tracing/trace_explorer/trace_view.md

{{% /tab %}}
{{% tab "Map" %}}

> {{< img src="tracing/trace_view/trace-map.png" alt="Trace map" style="width:100%;">}}

Reviewer: The image is broken, is this a new image that needs to be included?

Reviewer: Thanks @mervebolatdd for the links! The image wasn't pulled into this branch, so I merged master into your branch. The image is there now.

> Trace map displays a representation of all services involved in a single trace, providing an overview of the transaction lifecycle at the service level and clarifying service dependencies.

Reviewer: ```suggestion
Trace map displays a representation of all services involved in a single trace. It provides an overview of the transaction lifecycle at the service level and shows service dependencies.
```

> Each node on the map represents a service in the transaction lifecycle. To prevent cyclic dependencies on the map, services that call back to a service already invoked by them are represented by duplicated nodes.

Reviewer: ```suggestion
Each node on the map represents a service in the transaction lifecycle. To prevent cyclic dependencies on the map, services that call another service that had already been invoked by the original service, are represented by duplicated nodes.
```

> Service nodes explicitly show the percentage of the **total execution time** which provides an overview of the trace duration breakdown at the service level.

Reviewer: What is the percentage of? What is the "total execution time" being compared to?

```suggestion
Service nodes explicitly show the percentage of the **total execution time**, which shows the trace duration breakdown at the service level.
```

> If a [service entry span][6] is in an error state, the corresponding service node is marked with a red border to highlight faulty services at a glance. If an error occurs in a service exit span, the edge indicating the call to the next service is also highlighted in red.

Reviewer: ```suggestion
If a [service entry span][6] is in an error state, the corresponding service node is marked with a red border to highlight a faulty services. If an error occurs in a service exit span, the edge indicating the call to the next service is also highlighted in red.
```

> To view additional information about the service entry spans for each node, hover over the error state. The tooltip displays details about the service entry span's operation and resource name along with any error message. For further investigation, you can switch to the Waterfall view using the ``View Entry Span`` button.

Reviewer: ```suggestion
To view additional information about the service entry spans for each node, hover over the error state. The tooltip displays details about the service entry span's operation and resource name, along with any error messages. To further investigation, click **View Entry Span** to switch to the Waterfall view.
```
